sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -49,6 +50,13 @@ export interface GroupCloseAllConnectionsOptions extends OperationOptions {
 export interface GroupHasUserOptions extends OperationOptions {
 }
 
+// @public
+export interface GroupListConnectionsOptions extends OperationOptions {
+    continuationToken?: string;
+    maxPageSize?: number;
+    top?: number;
```

Comment:
Do we want to expose `top` too?  Most of our paging operations just take page size and continuation tokens.

Reply:
`top` sometimes is useful in our scenario. For example, there are too many connections in a group, and the callers only want a few of them for displaying.